src/js/controllers/index.js

```diff
@@ -782,6 +735,114 @@ angular.module('copayApp.controllers').controller('indexController', function($r
     });
   };
 
+  self.stopSync = function(remoteTx, localTx) {
+    if (remoteTx.txid == localTx.txid)
+      return true;
+    else
+      return false;
+  }
+
+  self.removeLessThanSixConfirmations = function(txs) {
+    return lodash.map(txs, function(tx) {
+      if (tx.confirmations >= CONFIRMATIONS)
+        return tx;
+    });
+  }
+
+  self.updateLocalTxHistory = function(cb) {
+    var fc = profileService.focusedClient;
+    var c = fc.credentials;
+    var txsToPush = [];
+
+    storageService.getTxHistory(c.walletId, function(err, txs) {
```

Comment:
it seems that from 757 to 771 could be a new function:
`self.getLocalTxs(function(err, localTxs) {`